ingestor/src/main/scala/uk/ac/wellcome/ingestor/controllers/ManagementController.scala

```diff
@@ -0,0 +1,16 @@
+package uk.ac.wellcome.platform.ingestor.controllers
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import com.twitter.finagle.http.Request
+import com.twitter.finatra.http.Controller
+
+
+@Singleton
+class ManagementController @Inject()() extends Controller {
+  get("/management/healthcheck") { request: Request =>
+    response.ok.json(Map("message" -> "ok"))
```

Comment:
I think there should be a `/management/manifest` endpoint for each application that carries:

- Name
- Version

Like this:

https://github.com/guardian/grid/blob/master/common-lib/src/main/scala/com/gu/mediaservice/lib/management/Management.scala#L18

Also extractable to a common library.